tests/plugins/easyimage/manual/imageoutline.html

```diff
@@ -0,0 +1,43 @@
+<h2>Classic editor</h2>
```

Comment:
Please, think a little about developer experience here.

Currently it looks like that:

![Small editor containg a widget that can not seen as a whole](https://user-images.githubusercontent.com/5353898/34937378-e70beed8-f9e4-11e7-81fb-4fed38ef573a.png)

And you're asking tester to examine whether whole widget looks correctly - but the widget itself can not be viewed in editor as a whole.

You could improve it in several ways:
* Change the image displayed to the one with less height. [`tests/plugins/image2/_assets/bar.png`](https://github.com/ckeditor/ckeditor-dev/blob/t/932-2961-3302/tests/plugins/image2/_assets/bar.png) seems to be a much better choice.
* Increase height of the editor - less elegant and tester friendly solution.